### ISSUE TITLE
forum.doktoronline.no

### ISSUE THREAD

**Issue by DandelionSprout** [finnish-easylist-addition/finnish-easylist-addition#305]
### List the website(s) you're having issues:

```
https://forum.doktoronline.no/
https://forum.kvinneguiden.no/
```

### What happens?

I think having `##[class^="dfp"]` as a generic entry in Adblock List for Finland, is a very bad idea due to the entry being too short, and thus very broadly blocking. In this specific case, it causes `https://forum.doktoronline.no/` to be completely blanked out.

It should either be removed entirely, made domain-specific, or at a bare minimum be changed to `##[class^=dfp-]:not(html):not(body)`

### List Subscriptions you're using:

![image](https://user-images.githubusercontent.com/22780683/124694725-20d90200-dee2-11eb-8cb0-a5451ad9fbd2.png)

### Your settings

- OS/version: Windows 10 21H1 x64
- Browser/version: Chrome 91.0.4472.124 x64
- Adblock Extension/version: uBO 1.36.1b2 + I Don't Care About Cookies 3.3.1

### Other details:

![image](https://user-images.githubusercontent.com/22780683/124694859-5e3d8f80-dee2-11eb-8b6c-6f1e29ab0717.png)

**Comment by peace2000** [comment]
Quickfixed https://github.com/finnish-easylist-addition/finnish-easylist-addition/commit/31d555504c48db5bf8edfbc1c877a3cad78e09df gonna come back to this later.

**Comment by peace2000** [comment]
The reason behind this rule was that I saw in many Finnish websites ad elements that started with _dfp_ so I thoght I would be some kind more widely used identifier for ad elements. The rule was added when I was processing this issue report https://github.com/finnish-easylist-addition/finnish-easylist-addition/issues/295

But anyway, this was a bad call as it caused issues.

**Comment by Nojuuu** [comment]
> I thought it would be some kind even more widely used identifier for ad elements. 

dfp usually stands for "Doubleclick for Publishers", and is quite common in ad elements, but there doesn't seem to be any general rule for naming those elements, so devs probably just put whatever they want. 

But yeah, having just dfp seems to be too short identifier.

**Comment by peace2000** [comment]
https://github.com/finnish-easylist-addition/finnish-easylist-addition/commit/b8f8b508dc759f257011c973774df70981599bec

Should be safe now :)

**Comment by peace2000** [comment]
Interestingly the element name _dfp-loaded_ seems to be used on many Norwegian websites: https://publicwww.com/websites/%22dfp-loaded%22/

**Comment by peace2000** [comment]
@DandelionSprout You might want to apply this Ubo-rule for adblocking purposes, for Norwegian websites: `*##+js(rc, dfp-loaded, html)`

I have noticed that removing attributes from elements will affect the content of the website. For e.g. I have managed to fix some complex cookie banner websites by just removing the cookie banner related attribute from html or body elements.

I don't know how familiar you are with that scriplet, but these pictures clarify what it removes and from where:

![kuva](https://user-images.githubusercontent.com/17256841/126790738-ccce6c87-33ac-43dd-866e-85f3af1c1374.png)

![kuva](https://user-images.githubusercontent.com/17256841/126790826-c85b6b0e-bff7-45f8-a3b0-b789a8e94fa4.png)

https://krisssy.blogg.no/

(This would work as well: `*##+js(ra, class, .dfp-loaded` but I feel it's riskier as it removes the whole attribute, the first rule removes just one specific value, making it safer for generic use.)
